src/commonMain/kotlin/org/kosat/CDCL.kt

```diff
@@ -48,96 +80,70 @@ class CDCL(val clauses: MutableList<MutableList<Int>>, initNumber: Int = 0) {
         }
     }
 
-    data class VarState(
-        var status: VarStatus,
-        var clause: Int,
-        var level: Int,
-    )
+    // TODO why not abs..
+    private fun litIndex(lit: Int): Int = abs(lit)
```

Comment:
WHY it is called "`litIndex`"?..

Okay, I can come up with an explanation, but it is not very straightforward. Anyway, add some docs, and leave it for now.